src/services/auth/index.ts

```diff
@@ -0,0 +1,2 @@
+export const Auth = true;
+
```

Comment:
you entered two empty lines, please remove this one and always make sure how your code looks in the diff inspection of GitHub